docs/variants/versions.md

```diff
@@ -0,0 +1,29 @@
+# Version matrix
+
+## About
+
+The table below contains an up to date list of all supported platforms,
+including latest supported coreboot and EDK2 versions and branches from which
+you can build firmware for your platform.
+
+| Platform                | coreboot branch                                                                                       | Dasharo base                                                          | coreboot version | EDK2 version | EDK2 rev                                                                                    |
```

Comment:
I think that `coreboot version` should be removed. It is strictly correlated with `Dasharo base`, and dropping this column will make more space for the rest.

Reply:
Done